<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,
    initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="AnimLogo.css">
    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
  </head>
  <body>
    <header class="header">
      <div class="logo-container">
        <img src="Logo.jpeg" class="logo" alt="Logo">
        <span class="year-text">2024</span>
      </div>
      <div class="navbar">
        <div class="dropdown">
          <a href="Introduction">INTRODUCTION</a>
          <div class="dropdown-content">
            <a href="Introduction.html#background">BACKGROUND</a>
            <a href="Introduction.html#problem">PROBLEM</a>
            <a href="Introduction.html#objectives">OBJECTIVES</a>
            <a href="Introduction.html#goals">GOALS</a>
          </div>
        </div>
        <div class="dropdown">
          <a href="Design">DESIGN</a>
          <div class="dropdown-content">
            <a href="Design.html#metabolite-channel-intro">METABOLITE CHANNELLING OVERVIEW</a>
            <a href="Design.html#nanotube-structure-design">NANOTUBE STRUCTURE DESIGN</a>
          </div>
        </div>
        <div class="dropdown">
          <a href="Build">BUILD</a>
          <div class="dropdown-content">
            <a href="#enzyme-synthesis">IsPETase AND IsMHETase SYNTHESIS</a>
            <a href="#dna-origami">DNA ORIGAMI</a>
          </div>
        </div>
        <div class="dropdown">
          <a href="Model">MODEL</a>
          <div class="dropdown-content">
            <a href="Model.html#diffusion-modelling">SIMULATION</a>
          </div>
        </div>
        <a href="Test">TEST</a>
        <a href="FutureWork">FUTURE WORK</a>
        <a href="TheTeam">THE TEAM</a>
      </div>
    </header>

    <main id="content" class="main-content" role="main">
        <a href="https://imperialbiomod24.github.io/" class="image-link">
          <div class="image-container">
            <img src="Logo_1.svg" alt="Logo-1" class="logo-1-image">
            <img src="Logo_2.svg" alt="Logo-2" class="logo-2-image">
            <img src="Arrow.svg" alt="Arrow Image" class="logo-image">
          </div>
        </a>

        <div class="main-build-container">
          <div class="build-container">
            <h1 class="build-main-title">Experimental Work<br><br></h1>
            <div id="enzyme-synthesis" class="section build-sect-1">
              <span class="build-title-1">IsPETase AND IsMHETase SYNTHESIS</span>
              <div class="divider"></div>
              <p class="build-sect-1-body">The gene expression sequences for Ideonella Sakaiensis PETase and MHETase were found
                from previous gene expression studies (<a href="#ref1">1</a>,<a href="#ref2">2</a>). PETase was expressed with a lamB signal peptide from maltoporin 
                protein, promoting a Sec-dependent pathway and localisation to the outer e.coli membrane as this has been shown
                to increase secretion and yield of protein, <a href="#ref3">3</a>. Similar studies could not be found for IsMHETase, thus the protein
                was expressed with its native signal peptide as described in Janatunaim & Fibrani’s 2020 study.<br><br>
              </p>
              <span class="build-title-2">MATERIALS AND METHODS</span>
              <div class="divider"></div>
              <div class="build-sect-2-body">
                <ul-1>
                  <li>Amino acid sequences were checked against <a href="https://www.rcsb.org" target="_blank">PDB sequences</a> (6ILW for PETase, 6QZ4 for MHETase), using <a href="https://en.vectorbuilder.com/tool/sequence-alignment.html" target="_blank">
                  Vector Builder Sequence Alignment Tool</a>.
                  As the PETase sequence had already been codon-optimised in the Joo. et al 2018 study, only MHETase required codon optimisation which was
                  performed with the <a href="https://eu.idtdna.com/pages/tools/codon-optimization-tool" target="_blank"> IDT Biosciences codon optimisation tool</a>.<br><br></li>
                </ul-1>

                <ol class="ol-2">
                  <li>A pUCIDT +Kan plasmid with each gene of interest were ordered from IDT Biosciences on glycerol stock.</li>
                  <li>Plasmids were spread on LB agar and incubated at 37ºC overnight.</li>
                  <li>Two colonies were selected from each gene plate and incubated individually in 5ml LB broth with Kanamycin, with vigorous shaking at 37ºC overnight.</li>
                  <li>A miniprep was then performed on each of the 4 samples before DNA extract was added to 4 identical liquid E. coli BL21 DE3 cultures and spread over LB agar with Kanamycin and cultured at 37ºC overnight.</li>
                  <li>One plate for each gene was successful; one colony from each successful plate was then cultured in 100ml LB broth with Kanomycin, with vigorous shaking at 37ºC overnight.</li>
                  <li>Each culture was then made up to 1L and grown to an optical density of 0.6 and 0.8, after approximately 90 minutes.</li>
                  <li>Both proteins were than purified as described in <a href="#ref3">3</a>, after which assay with SDS-PAGE was performed to confirm protein production.</li><br><br>
                </ol>
              </div>
            </div>
            <div id="dna-origami" class="section build-sect-3">
              <span class="build-title-3">DNA ORIGAMI</span>
              <div class="divider"></div><br>
              <span class="build-title-3-sub1">SYNTHESIS</span>
              <p class="build-sect-3-body-1">
                The synthesis and assembly of the DNA origami PETase and MHETase module units comprising the cascade was adapted from Linko et al. <a href="#ref4">4</a>.<br><br>
                Staple sequence strands were purchased from IDT. For each of the PETase and MHETase module units, 3 staple strands contained a C<sub>6</sub>-amine modification (5AmMC6)
                on the 5’ end, intended to act as enzyme linkers for downstream enzyme immobilisation. All strands were first centrifuged (2 mins, 3000 xg) and subsequently resuspended
                in an amount of 1x TE buffer (10mM pH 8.0 Tris-HCl, 0.1mM EDTA; filter-sterilised 0.22µm filter) to achieve a final concentration of 100&micro;M. 
                Resuspension was achieved by a combination of centrifugation (2 mins, 3000 xg) and vortexing (1000 rpm) to ensure homogeneity.
                Oligonucleotides were then stored at -20◦C to maximise stability <a href="#ref5">5</a>.<br><br>

                Concentration was then verified by measuring the absorbance at 260nm (A<sub>260</sub>) using the NanoDrop&trade; One Spectrophotometer, assuming a standard ssDNA conversion
                factor of 33&micro;g/A<sub>260</sub>, where 
                \[
                \text{Concentration}, c \, [\mu\text{g/mL}] = A_{260} \times \text{Conversion Factor}
                \]
                and using the Beer-Lambert Law,
                \[
                A_{260} = \varepsilon_{260} \times c \times p
                \]
                where
                \[
                \varepsilon_{260}
                \]
                is the extinction coefficient [L/mole.cm] and
                \[
                \text{p}
                \]
                is the length of light path through sample [cm]. Given some discrepancies between the expected and measured oligo concentration, this was in part likely due to the
                conversation factor being oligo-specific, which is known to vary between sequence context and base composition <a href="#ref6">6</a>.<br><br>
              </p>.

              <span class="build-title-3-sub2">ASSEMBLY</span>
              <p class="build-sect-3-body-2">
                To synthesise and verify assembly of each module unit, staple strands (including unmodified enzyme linker strands) corresponding to
                that of the PETase and MHETase structures were mixed and diluted to achieve a final concentration of 500nM.<br><br>
                For PET Module Unit:<br>
                <ul-1>
                  <li>5&micro;L of each staple strand (total: 183 strands), topped up with 85&micro;L Ultrapure water</li>
                  <li>5&micro;L of each staple strand (total: 187 strands), topped up with 65&micro;L Ultrapure water</li>
                </ul-1>
                <br><br>
                To test for the optimal Mg<sup>2+</sup> ion concentration to achieve optimal folding, DNA origami was folded under
                <a href="#table-conditions">4 different conditions</a>, with varying concentration of MgCl<sub>2</sub>.<br>
              </p>
              <figure>
                <img src="Table1.png" alt="Conditions Table" style="width:75%;">
                <figcaption id="table-conditions">
                  Table 1 - 4 sample conditions with varying MgCl<sub>2</sub> concentration.<br>
                  Note that R<sub>20</sub> has a higher staples:scaffold molar ratio (20:1), relative to all other conditions (10:1).<br><br>
                </figcaption>
              </figure>
              <p class="build-sect-3-body-2">
                Each condition was made by mixing an amount of scaffold, staples, and folding buffer (1x TAE buffer (40mM pH 8.0 Tris-HCl,
                19mM acetic acid, 1mM EDTA),  MgCl<sub>2</sub>, NaCl) together to achieve <a href="#table-conditions-all">50&micro;L of DNA solution</a>.<br>
              </p>
              <figure>
                <img src="Table2.png" alt="Conditions Table All" style="width:75%;">
                <figcaption id="table-conditions-all">
                  Table 2 - Composition of each DNA solution.<br><br>
                </figcaption>
              </figure>
              <p class="build-sect-3-body-2"></p>
                Each DNA origami solution was then annealed over 2 folding temperature ramps (75 minutes, 65-60◦C; 57 hrs, 59-40◦C).<br><br>

                Agarose gel electrophoresis was then conducted to verify the quality of and purify the folded origami. Samples were run in
                1% agarose gel, prepared by mixing 1g agarose with 100mL 0.1x TAE buffer, and then microwaved for 3 minutes.
                Once cooled, 30µL of Ethidium bromide solution (0.0625&micro;g/L) was added. The gel was then cast into a gel box with combs.
                For use as a reference, M13mp18 scaffold strands (7249 bp) were diluted to 50nM, and 1kb DNA ladders were diluted 1 in 10, both
                with 0.1x TAE buffer. 5µL of each DNA solution, scaffold and DNA ladder were added to 1&micro;L of 6X Purple Loading Dye.
                The gel was run for 40 minutes at 120V. Gel images were then captured by a Gel Doc (iBright&trade; CL750 Imaging System) [ref figure in test section].
              </p>
            </div>
          </div>

          <div class="references-container">
            <div class="section references-sect">
              <span class="references">REFERENCES</span>
              <div class="divider"></div>
              <p class="references-body">
                <a id="ref1"><strong>1.</strong>	Joo S. et al. Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation. Nat Commun. 2018 Jan 26;9(1):382</a><br><br>
                <a id="ref2"><strong>2.</strong>	Janatunaim RZ, Fibriani A. Construction and Cloning of Plastic-degrading Recombinant Enzymes (MHETase). BIOT. 2020 Sep 25;14(3):229–34</a><br><br>
                <a id="ref3"><strong>3.</strong>	Seo H. et al. Production of extracellular PETase from Ideonella sakaiensis using sec-dependent signal peptides in E. coli. Biochemical and Biophysical Research Communications. 2019 Jan 1;508(1):250–5</a><br><br>
                <a id="ref4"><strong>4.</strong>	Linko V, Eerikäinen M, A. Kostiainen M. A modular DNA origami-based enzyme cascade nanoreactor. Chemical Communications. 2015;51(25):5351–4</a><br>
                <a id="ref5"><strong>5.</strong>  Podivinsky E. et al. Effect of storage regime on the stability of DNA used as a calibration standard for real-time polymerase chain reaction. Analytical Biochemistry. 2009 Nov 1;394(1):132-4</a><br><br>
                <a id="ref6"><strong>6.</strong>  Kara&ccedil;ay B. Four tips for oligonucleotide quantification with Thermo Scientific&trade; NanoDrop&trade; instruments. IDT. 2018 Jan 29</a><br><br>
              </p>
            </div>
          </div>
        </div>
    </main>

  </body>
</html>

<!-- Possibly need to import jekkel fonts into here? - other stuff from default.html? --> 